src/client/ClientGameEngine.cpp

```diff
@@ -16,302 +17,343 @@
 
 namespace client
```

Comment:
Put the indent back, I will do a PR by this morning to do that in a proper way with `using`